librz/core/cmd/cmd_print.c

```diff
@@ -6124,7 +6124,70 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
 				}
 				rz_cons_printf(".equ shellcode_len, %d\n", len);
 			} else {
-				rz_print_code(core->print, core->offset, core->block, len, input[1]);
+				char *str = NULL;
+				bool big_endian = rz_config_get_b(core->config, "cfg.bigendian");
+				// rz_print_code(core->print, core->offset, core->block, len, input[1]);
+				switch (input[1]) {
+				case '*': // "pc*" // rizin commands
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_RIZIN);
+					break;
+				case 'a': // "pca" // GAS asm
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_ASM);
+					break;
+				case 'S': // "pcS" // bash shellscript
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_BASH);
+					break;
+				case 'J': // "pcJ" // nodejs
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_NODEJS);
+					break;
+				case 'G': // "pcG" // golang
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_GOLANG);
+					break;
+				case 'k': // "pck" kotlin
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_KOTLIN);
+					break;
+				case 'z': // "pcz" // swift
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_SWIFT);
+					break;
+				case 'r': // "pcr" // Rust
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_RUST);
+					break;
+				case 'o': // "pco" // Objective-C
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_OBJECTIVE_C);
+					break;
+				case 'v': // "pcv" // java
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_JAVA);
+					break;
+				case 'V': // "pcV" // vlang.io
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_VLANG);
+					break;
+				case 'y': // "pcy" // yara
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_YARA);
+					break;
+				case 'j': // "pcj" // json
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_JSON);
+					break;
+				case 'P': // "pcP" // python
+				case 'p': // "pcp" // python
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_PYTHON);
+					break;
+				case 'h': // "pch" // C half words with asm
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, big_endian ? RZ_LANG_BYTE_ARRAY_C_CPP_HALFWORDS_BE : RZ_LANG_BYTE_ARRAY_C_CPP_HALFWORDS_LE);
+					break;
+				case 'w': // "pcw" // C words with asm
+					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, big_endian ? RZ_LANG_BYTE_ARRAY_C_CPP_WORDS_BE : RZ_LANG_BYTE_ARRAY_C_CPP_WORDS_LE);
```

Comment:
split the line?

Reply:
tbh, that is old shell. i would not care too much